.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,6 @@
 
 **Tested scenarios**
 <!-- Description of tested scenarios -->
-<!-- Please verify that the unit tests are passing by running "vendor/bin/phpunit -c ." -->
+<!-- Please verify that the unit tests are passing by running "vendor/bin/phpunit -c dev/tests/unit/phpunit.xml.dist vendor/adyen/module-payment/Test/" -->
```

Comment:
then let's remove this as well https://github.com/Adyen/adyen-magento2/blob/10f2a982c9e6a33e840e4434188928856a27fb3a/phpunit.xml

Reply:
is removed